app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number):
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
+            cleaning_staff: Cleaner
```

Comment:
But now it is relevant to write a comma after the last item in the column
```suggestion
            cleaning_staff: Cleaner,
```